ckanext/security/validators.py

```diff
@@ -15,17 +14,12 @@
 
 
 def user_password_validator(key, data, errors, context):
-    # joeg: somehow key is a tuple: ('password1', ), but value is a string
-    field = key[0]
     value = data[key]
 
     if isinstance(value, Missing):
         pass  # Already handeled in core
     elif not isinstance(value, basestring):
-        raise ValidationError(
-            {field: [_('Passwords must be strings.')]},
-            error_summary={_('Password'): _('invalid new password')}
-        )
+        raise Invalid(_('Passwords must be strings.'))
```

Comment:
Ok. 
But the args not setting error=summary is that part of what you are rechecking?